examples/fluids/index.rst

```diff
@@ -293,27 +293,12 @@ different wind types.
    In this case, a background wind with a constant velocity field, enters the domain and compels
    the blob of total energy to leave the domain.
 
-   For the inflow boundary conditions, we have followed :cite:`kyncl2017riemann` which offers solution
-   to a modified Riemann problem by preference of pressure (:math:`P`). We have chosen the pressure on
-   the boundary to be equal to the one inside the domain, so the incoming nonlinear wave has no amplitude.
-   The boundary values are computed as following
-
-   .. math::
-
-      \begin{aligned}
-         P_* &= P_R\\
-         u_* &= u_R + \frac{2}{\gamma - 1} \sqrt{\gamma \frac{P_R}{\rho_R} } \left[ 1
-         - \left( \frac{P_*}{P_R} \right)^\frac{\gamma - 1}{2 \gamma} \right]
-      \end{aligned}
-
-   where the subscripts :math:`*`, :math:`R`, and :math:`L` denote the values on the boundary, inside
-   the domain, and incoming wind, respectively. The computed variables :math:`P_*` and :math:`u_*` and,
-   a prescribed :math:`E_L` are applied weakly on the inflow boundaries.
-
-   For the outflow boundary conditions, we have
-   applied values from previous iteration, following :cite:`papanastasiou1992outflow` which extends the
-   validity of the weak form of the governing equations to the synthetic outflow instead of replacing them
-   with unknown essential or natural boundary conditions.
+   For the inflow boundary conditions, a prescribed :math:`E_wind` is applied weakly on the inflow boundaries.
+
+   For the outflow boundary conditions, we have applied values from previous iteration, following
```

Comment:
Thanks Leila for adding this! It looks great! I'm having a bit of a hard time to grasp this sentence, where we refer to previous interations to calculate a spatial flux. Is this what this sentence means?

```suggestion
   In order to calculate :math:`\bm{F}(\bm q_N^{n+1})`, for the outflow boundary condition, we have used values of :math:`E` from the previous time iteration, namely  :math:`E^n`, following
```

Reply:
Unless I'm missing context, I don't think it's true that we use values from the previous time step.

Reply:
Sorry for the confusion.
I guess I wrote this because I was only thinking of explicit time stepping. 

How about: 
"For the outflow boundary conditions, we have used the current values of `E`, following"
?

Reply:
I think it's all consistent with the semi-discrete form `g(t, q, qdot) = 0` or `qdot = f(t, q)` in that all state variables appear at the same instant. (Stages in the time integrator correspond to quadrature points in time.) So we're quite literally performing the integral that appears in the weak form.